### PR TITLE
Fixes #28992 - Add description to plugin roles

### PR DESCRIPTION
**Pull request by stejskalleos**
PR is part of  [Add default descriptions to all roles added from plugins](https://projects.theforeman.org/issues/28936) task.

### CODE REVIEW

**Comment by stejskalleos** [comment]
Hi @timogoebel , can you please give me some good description for the roles? Thanks

**Comment by timogoebel** [comment]
Does this require a minimum Foreman version?

File: lib/foreman_omaha/engine.rb [hunk 1]
```diff
@@ -49,9 +49,11 @@ class Engine < ::Rails::Engine
         end
 
         role 'Omaha reports viewer',
-             [:view_omaha_reports]
+             [:view_omaha_reports],
+             'TODO: Description'
```
Comment (timogoebel) [inline lib/foreman_omaha/engine.rb]: This role can view Omaha reports.

File: lib/foreman_omaha/engine.rb [hunk 2]
```diff
@@ -49,9 +49,11 @@ class Engine < ::Rails::Engine
...
         role 'Omaha reports manager',
-             [:view_omaha_reports, :destroy_omaha_reports, :upload_omaha_reports]
+             [:view_omaha_reports, :destroy_omaha_reports, :upload_omaha_reports],
+             'TODO: Description'
```
Comment (timogoebel) [inline lib/foreman_omaha/engine.rb]: This role can manage Omaha reports.

**Comment by stejskalleos** [comment]
> Does this require a minimum Foreman version?

No it doesn't, it's not a new feature.

**Comment by timogoebel** [comment]
@stejskalleos: Great. Is this ready to review (& merge)? The title still shows the WIP label.

**Comment by stejskalleos** [comment]
Sorry, forgot to update title. It's ready to merge, if nobody else has any other comments or ideas.

**Comment by stejskalleos** [comment]
Hi @timogoebel,
I just found that role descriptions actually requires minimal version of foreman 1.20+.

File: lib/foreman_omaha/engine.rb [hunk 3]
```diff
@@ -49,9 +49,11 @@ class Engine < ::Rails::Engine
         end
 
         role 'Omaha reports viewer',
-             [:view_omaha_reports]
+             [:view_omaha_reports],
+             'This role can view Omaha reports.'
```
Comment (timogoebel) [inline lib/foreman_omaha/engine.rb]:
```suggestion
             'Role granting permissions to view Omaha reports.'
```

File: lib/foreman_omaha/engine.rb [hunk 4]
```diff
@@ -49,9 +49,11 @@ class Engine < ::Rails::Engine
...
         role 'Omaha reports manager',
-             [:view_omaha_reports, :destroy_omaha_reports, :upload_omaha_reports]
+             [:view_omaha_reports, :destroy_omaha_reports, :upload_omaha_reports],
+             'This role can manage Omaha reports.'
```
Comment (timogoebel) [inline lib/foreman_omaha/engine.rb]:
```suggestion
             'Role granting permissions to manage Omaha reports.'
```

**Comment by stejskalleos** [comment]
[test foreman_omaha]

**Comment by timogoebel** [comment]
Thanks, @stejskalleos! And sorry for the delay!

**Pull request merged**